### DISCUSSION TITLE
Is this a bug? Daily puzzle 22/5/2024

### DISCUSSION THREAD

**Discussion by nzkks** [cssbattle/forum#391] [General]
![image](https://github.com/cssbattle/forum/assets/6157598/b926890f-ccaf-4942-967a-4450a23c16fa)
I don't understand, why I got the low score and only 90.9% match. Some one please tell me. Thanks

My code below:
`<p><dl><dt><style>*{background:#F48B26}p,dl{background:#FEFF58;width:440;height:110}p{margin:-8 0 0 32;border-bottom:dotted 43q#F48B26}dl{margin:0 0 0-88;border-top:dotted 43q#F48B26}dt{margin:-60;height:40`

**Comment by nzkks** [comment]
After reading an answer to similar issue, I shouldn't have used Firefox or non-chromium browsers.

**Reply by Tryskall** [reply]
I ve tried it on chrome firefox and safari its still only gives me 99.3%
